test/unit/test_classical_explainer.py

```diff
@@ -45,3 +47,23 @@ def test_explain_model_local_with_predicted_label(self):
         predicted_label = label_encoder.inverse_transform(y)
         local_explanation = explainer.explain_local(DOCUMENT, predicted_label)
         assert len(local_explanation.local_importance_values) == len(local_explanation.features)
+
+    def test_pickle(self):
+        """
+        Test for pickling of classical explainer
+        :return:
+        """
+        X_train, X_test, y_train, y_test = setup_mnli_test_train_split()
+        label_encoder = LabelEncoder()
+        y_train = label_encoder.fit_transform(y_train)
+        explainer = ClassicalTextExplainer()
+        classifier, best_params = explainer.fit(X_train, y_train)
+        explainer_file_name = 'explainer.p'
+        with open(explainer_file_name, 'wb') as explainer_save_file:
```

Comment:
Often it's safer to use a tempfile and pytest has supported fixtures: https://docs.pytest.org/en/stable/tmpdir.html.
This will just help long term to save space on build agents if tests fail before cleanup happens

Reply:
done!